### PR TITLE
Jormungandr: use speed in experimental

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

File: source/jormungandr/tests/routing_tests_experimental.py [hunk 1]
```diff
@@ -74,37 +74,45 @@ def check_previous_datetime_link(dt, response, clockwise):
...
-        super(JourneyCommon, self).test_sp_to_sp()
+        super(TestJourneysExperimental, self).test_sp_to_sp()
 
-    @skip("temporarily disabled")
     def test_speed_factor_direct_path(self):
-        super(JourneyCommon, self).test_speed_factor_direct_path()
+        """
+        This feature is no longer supported
+        """
+        pass
```
Comment (TeXitoi) [inline source/jormungandr/tests/routing_tests_experimental.py]: Looking at the title of the test, you should just use another way to get the direct path that don't use type filter. Using max_duration=0 should do the trick

File: source/jormungandr/tests/routing_tests_new_default.py [hunk 2]
```diff
@@ -75,25 +75,35 @@ def check_previous_datetime_link(dt, response, clockwise):
...
+        """
+        pass
 
-    @skip("temporarily disabled")
     def test_speed_factor_direct_path(self):
-        super(JourneyCommon, self).test_speed_factor_direct_path()
+        """
+        This feature is no longer supported
+        """
+        pass
```
Comment (TeXitoi) [inline source/jormungandr/tests/routing_tests_new_default.py]: same as in experimental.

**Review by TeXitoi — CHANGES_REQUESTED** [review]

File: source/jormungandr/tests/routing_tests_experimental.py (hunk 1 above)
Comment (xlqian) [reply source/jormungandr/tests/routing_tests_experimental.py]: you're right, this test shouldn't be overwritten...

File: source/jormungandr/tests/routing_tests_new_default.py (hunk 2 above)
Comment (xlqian) [reply source/jormungandr/tests/routing_tests_new_default.py]: ok

**Review by TeXitoi — APPROVED** [review]

**Comment by TeXitoi** [comment]
It should fix a lot of artemis tests.

**Pull request merged**